### DISCUSSION TITLE
[Feature] Show frames with names around people's faces

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8677] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If faces are detected in the photo, they are visible in the details pane. We can hover over the thumbnail of that person and a frame around the detected face will be visible directly on the photo.

That functionality is great, but I would suggest expanding it a little bit by keeping existing functionality and adding reversed behavior as well.

What I mean is when I move the cursor anywhere in the photo area, then all frames around the faces should be slightly visible (reduced opacity) to signal where are recognized face regions. Then when I move the cursor to the place where there is a detected face that frame should change to full opacity and the name of a person should be visible under that frame.

When I move the cursor out of the photo section, then none of the frames should be visible at all.

If we have a photo with several or even more people and someone asks who it is or we want to see more photos of that specific person, it's easier to point at that person directly on the photo instead of looking for him/her in a list of people in the details pane.

Here is an example of what I mean exactly:

![face_frame](https://github.com/immich-app/immich/assets/28065743/44cf7caa-a300-4f0d-8e61-35f963632cb0)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by chodthewacko** [comment]
This will also greatly help spotting if a face was ignored by face recognition in group pictures.

**Comment by bo0tzz** [comment]
#3489
